sqlalchemy_utils/types/scalar_list.py

```diff
@@ -50,13 +51,31 @@ class Player(Base):
         session.commit()
 
 
+    :param inner_type:
+        The type of the values. Default is ``str``.
+    :param separator:
+        Separator of the values. Default is ``,``.
+    :param coerce_func:
+        Custom function to coerce values when read from database.
+        By default ``inner_type`` is used instead.
     """
 
     impl = sa.UnicodeText()
 
-    def __init__(self, coerce_func=six.text_type, separator=u','):
+    def __init__(self, inner_type=six.text_type, separator=u',',
```

Comment:
Why not keeping the parameter order as it was and adding the new parameter as the last parameter? That way we wouldn't need to throw any warnings.

Reply:
I was thinking about that. I believe, having `inner_type` defined is the only way I can ensure validation of the values before they're written to database. So I made `inner_type` primary and `coerce_func` optional. It would be weird to have an optional parameter first.

Reply:
@kvesteri Any decision?

Reply:
@kvesteri I'd like to move this forward. Can I keep the change, or should I add the `inner_type` as the last parameter?